src/components/Personalization/groupDecisions.js

```diff
@@ -11,6 +11,7 @@ governing permissions and limitations under the License.
 */
 
 import { isNonEmptyArray, includes } from "../../utils";
```

Comment:
Same here, any reason we can't have `isPageWideSurface` in `./utils`.

Reply:
Yes, see previous comment.